### ISSUE TITLE
Provide RVD testsuite using arquilian

### ISSUE THREAD

**Issue by otsakir** [RestComm/Restcomm-Connect#1147]
Create testsuite for RVD integration testing. It could be implemented either as a package inside existing restcomm.testsuite or as a separate maven module - restcomm.rvd.testsuite. 

This is issue is about setting up the testsuite and provide tests for very common operations. Once the setup is proven working, further github issues  will be added for all testable operations.

**Comment by ghjansen** [comment]
Maybe some info present in #747 can be useful here. :)

**Comment by otsakir** [comment]
I tried to use a single arquilian container (mss-tomcat-embedded-7) to host both restcomm and RVD. I provided a single test class with two methods annotated with @Deployment. But the container complains that:
```
org.mobicents.servlet.sip.catalina.SipDeploymentException: No app-name present in the sip.xml deployment descriptor or no SipApplication annotation defined
	at org.mobicents.servlet.sip.startup.SipContextConfig.checkSipDeploymentRequirements(SipContextConfig.java:217)
```
It seems only sip-enabled web archives can be used with the specific container. Thus, RVD is ruled out.

On the other hand having many different containers (one that supports sip for restcomm, and a regular tomcat for RVD) is not possible either as stated in the documentation:

http://arquillian.org/guides/getting_started/  - search for *Add more containers*

Decent, self-contained integration testing for both restcomm and RVD does not seem possible. I'll keep iterating on an RVD-only solution with wireMock to simulate calls to restcomm.
